packages/core-ethereum/src/ethereum.ts

```diff
@@ -209,8 +210,13 @@ export async function createChainWrapper(providerURI: string, privateKey: Uint8A
   }
 
   async function announce(multiaddr: Multiaddr): Promise<string> {
-    const populatedTx = await channels.populateTransaction.announce(multiaddr.bytes)
-    return (await sendTransaction(true, populatedTx, channels.announce, multiaddr.bytes)).hash
+    const populatedTx = await channels.populateTransaction.announce(
+      publicKey.toUncompressedPubKeyHex(),
+      multiaddr.bytes
```

Comment:
shall we explicitly strip out the peer id here?

Reply:
yeah